packages/remix/src/client/RemixClerkProvider.tsx

```diff
@@ -84,7 +84,7 @@ export function ClerkProvider({ children, ...rest }: RemixClerkProviderProps): J
   return (
     <ClerkRemixOptionsProvider options={mergedProps}>
       <ReactClerkProvider
-        navigate={(to: string) => awaitableNavigateRef.current?.(to)}
+        push={(to: string) => awaitableNavigateRef.current?.(to)}
```

Comment:
❓Don't we need to pass replace ?
```suggestion
      push={(to: string) => awaitableNavigateRef.current?.(to)}
      replace={(to: string) => awaitableNavigateRef.current?.(to, { replace:true })}
```